README.md

```diff
@@ -45,17 +41,19 @@ Next, let's install all the dependencies. These are all listed in `requirements.
 
 Run that command, and you'll have all the dependencies installed and will be ready to run the code.
 
-## Step 3: Build the ONNX Model
+## Step 3: Build the TorchScript Model
 
-This is as easy as running the following:
+Load and train a Sklearn LogisticRegression model using the Iris Data Set. Use Microsoft's Hummingbird.ml to convert the Sklearn model into a TorchScript model for loading into RedisAI.
+Run the `build.py` Python script to generate the `iris.pt` model file:
```

Comment:
Nitpick: I do each paragraph as a single line and let the words wrap. Line 46 and 47 could be one line.

Reply:
Fixed